projects/plugins/jetpack/extensions/blocks/latex/edit.js

```diff
@@ -0,0 +1,110 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import {
+	Button,
+	TextareaControl,
+	/* eslint-disable @wordpress/no-unsafe-wp-apis */
+	__experimentalHStack as HStack,
+	__experimentalSurface as Surface,
+	__experimentalVStack as VStack,
+	/* eslint-enable @wordpress/no-unsafe-wp-apis */
+	Placeholder,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { useState, useEffect, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import katex from 'katex';
+import { icon } from './icon';
+import 'katex/dist/katex.min.css';
+import './editor.css';
+
+const LatexPlaceholder = () => {
+	return (
+		<span className="jetpack-latex-rendered-placeholder">{ __( 'Write LaTex…', 'jetpack' ) }</span>
+	);
+};
+
+export default function Edit( { attributes, setAttributes, isSelected } ) {
+	const { latex = '' } = attributes;
+	const textareaRef = useRef();
+	const [ renderAreaRef, setRenderAreaRef ] = useState( null );
+	const dispatch = useDispatch();
+	const props = useBlockProps( { className: 'jetpack-latex' } );
+
+	useEffect( () => {
+		if ( renderAreaRef ) {
+			katex.render( latex, renderAreaRef, {
+				throwOnError: false,
+				displayMode: false,
+				output: 'html',
+			} );
+		}
+	}, [ latex, renderAreaRef ] );
+
+	if ( isSelected ) {
+		return (
+			<div { ...props }>
+				<Placeholder
+					label={ __( 'LaTeX Block', 'jetpack' ) }
+					instructions={ __( 'Enter LaTeX code below', 'jetpack' ) }
+					icon={ icon }
+				>
+					<VStack gap={ 4 } justify="flex-start" align="stretch">
+						<TextareaControl
+							__nextHasNoMarginBottom
+							label={ __( 'LaTeX Code', 'jetpack' ) }
```

Comment:
```suggestion
							label={ __( 'LaTeX code', 'jetpack' ) }
```

We're using sentence case rather than Title Case.